prov/gni/src/gnix_vc.c

```diff
@@ -599,6 +600,7 @@ static int __gnix_vc_hndl_conn_resp(struct gnix_cm_nic *cm_nic,
 	 */
 
 	vc->conn_state = GNIX_VC_CONNECTED;
+	vc->peer_id = peer_id;
```

Comment:
i added that line to help with debugging.  its not really used, although I see I was thinking of using it in ```_gnix_rma_post_irq``` at some point.